src/pybind/mgr/telemetry/module.py

```diff
@@ -403,8 +404,26 @@ def compile_report(self, channels=[]):
             }
 
             report['fs'] = {
-                'count': len(fs_map['filesystems'])
+                'count': len(fs_map['filesystems']),
+                'feature_flags': fs_map['feature_flags'],
+                'num_standby_mds': len(fs_map['standbys']),
+                'filesystems': [],
             }
+            num_mds = 0
+            for fsm in fs_map['filesystems']:
+                fs = fsm['mdsmap']
+                report['fs']['filesystems'].append({
+                    'max_mds': fs['max_mds'],
+                    'ever_allowed_features': fs['ever_allowed_features'],
+                    'explicitly_allowed_features': fs['explicitly_allowed_features'],
+                    'in': len(fs['in']),
+                    'up': len(fs['up']),
+                    'num_mds': len(fs['info']),
```

Comment:
This implicitly includes the number of standby-replay daemons which I was going to ask to include anyway. Would perhaps be a good idea to explicitly calculate that in the JSON though.